pkgs/by-name/co/convmv/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  perl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "convmv";
+  version = "2.05";
+
+  outputs = [
+    "out"
+    "man"
+  ];
+
+  src = fetchzip {
+    url = "https://www.j3e.de/linux/convmv/convmv-${finalAttrs.version}.tar.gz";
+    hash = "sha256-ts9xAPRGUoS0XBRTmpb+BlGW1hmGyUs+rQLyUEgiZ54=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [ perl ];
+
+  buildInputs = [ perl ];
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "MANDIR=${placeholder "man"}/share/man"
+  ];
+
+  checkTarget = "test";
+
+  # testsuite.tar contains filenames that aren't valid UTF-8. Extraction of
+  # testsuite.tar will fail as APFS enforces that filenames are valid UTF-8.
+  doCheck = !stdenv.isDarwin;
+
+  prePatch =
+    lib.optionalString finalAttrs.doCheck ''
+      tar -xf testsuite.tar
+    ''
+    + ''
+      patchShebangs --host .
+    '';
+
+  dontPatchShebangs = true;
+
+  meta = with lib; {
+    description = "Converts filenames from one encoding to another";
+    downloadPage = "https://www.j3e.de/linux/convmv/";
+    license = with licenses; [ gpl2Only gpl3Only ];
+    maintainers = with maintainers; [ al3xtjames ];
```

I'd like to suggest to install the manpage by default unless there are there good reasons not to. What are your thoughts on this, @al3xtjames?
```suggestion
    outputsToInstall = [
      "out"
      "man"
    ];
    maintainers = with maintainers; [ al3xtjames ];
```

---

I'm not really familiar with`outputsToInstall`. Do we need to set it even if the only other output is `man`? https://github.com/NixOS/nixpkgs/blob/f1680774340d5443a1409c3421ced84ac1163ba9/pkgs/stdenv/generic/make-derivation.nix#L310-L320

---

Please disregard my recommendation, @al3xtjames. It appears that `outputsToInstall` is rarely used (about ~50 references of it are found in nixpkgs using `nix run nixpkgs#silver-searcher -- -l outputsToInstall | wc -l`) and it is likely going to be deprecated (see #255909).